examples/example-claim-fss.yaml

```diff
@@ -0,0 +1,14 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: demooci
+spec:
+  storageClassName: "oci-fss"
+  selector:
+    matchLabels:
+      oci-availability-domain: "PHX-AD-1"
```

Comment:
This is the older deprecated label. Can use `failure-domain.beta.kubernetes.io/zone: "PHX-AD-1"` instead.

Reply:
Fixed